### DISCUSSION TITLE
docker 方式安装后，http调用不返回消息

### DISCUSSION THREAD

**Discussion by alanOO7** [PaddlePaddle/PaddleOCR#13141] [Q&A]
docker 安装 paddleserving，安装后检测也通过了。docker内运行 python3 pipeline_http_client.py 也成功返回，但是通过接口调用的时候，没有返回结果。 请问这是啥原因。

接口 ：10.201.106.18:9292/ocr/prediction
参数：
{
    "key":["image"],
    "value":["图片的base64"]
}

**Comment by 350078238** [comment]
你的dockerfile发出来看下
